tests/testthat/_snaps/bin2factor.md

```diff
@@ -3,8 +3,8 @@
     Code
       prep(rec3, training = covers)
     Condition
-      Error in `prep()`:
-      ! The variables should be numeric
+      Error in `check_type()`:
```

Comment:
This will be part of the future PR to handle type checking 😄

Reply:
And I agree with the note. The error isn't great here.

This specific change came about because `step_bin2factor()` that used homegrown error checking that broke with the new changes https://github.com/tidymodels/recipes/pull/993/files/951dd0accb20b53f1d7fb94ff1089f0fbdfb8b76#diff-27fec350c23797c495bc728ce048bfe48844b512b737d869ed2303415e25cafdL94-R94